monitor_frontend/views/events_unnested_table.view.lkml

```diff
@@ -0,0 +1,120 @@
+include: "//looker-hub/monitor_frontend/views/events_unnested_table.view.lkml"
+
+view: +events_unnested_table {
+
+  dimension: event_extra__user_id {
+    description: "Unique user ID"
+    hidden: yes
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'user_id') ;;
+  }
+
+  dimension: event_extra__path {
+    description: "The URL or path of the page"
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'path') ;;
+    group_label: "Event Extra"
+    group_item_label: "Page Path"
+  }
+
+  dimension: event_extra__utm_campaign {
+    description: "Marketing and attribution UTM campaign"
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'utm_campaign') ;;
+    label: "UTM Campaign"
+  }
+
+  dimension: event_extra__utm_medium {
+    description: "Marketing and attribution UTM medium"
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'utm_medium') ;;
+    group_label: "Event Extra"
+    group_item_label: "UTM Medium"
+  }
+
+  dimension: event_extra__utm_source {
+    description: "Marketing and attribution UTM source"
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'utm_source') ;;
+    group_label: "Event Extra"
+    group_item_label: "UTM Source"
+  }
+
+  dimension: event_extra__utm_term {
+    description: "Marketing and attribution UTM term"
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'utm_term') ;;
+    group_label: "Event Extra"
+    group_item_label: "UTM Term"
+  }
+
+  dimension: event_extra__field_id {
+    description: "The location of an input field that determines where on the web page the interaction is occurring"
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'field_id') ;;
+    group_label: "Event Extra"
+    group_item_label: "Field ID"
+  }
+
+  dimension: event_extra__button_id {
+    description: "The text of the button that was clicked on, used as a way to identify where on the page the interaction is located"
```

Comment:
"_text of the button_" sounds like (potentially localized) words shown on the button, which I don't think is true.

Reply:
Copied this from a previous definition, but I'll remove until I verify a better version.

Reply:
Sorry, I didn't mean to dissuade you from having any description at all.  I think it'd be fine if "text" was simply changed to "ID".